main.tf

```diff
@@ -93,7 +93,7 @@ data "aws_iam_policy_document" "assume_role" {
 
     principals {
       type        = "Service"
-      identifiers = ["ec2.amazonaws.com"]
```

Comment:
Folks, can you please elaborate on why this needs to be a variable? I mean what other values can it be?
Just curious, as I can't think of a use-case.

Reply:
Hi, regarding our china account the value ["ec2.amazonaws.com"], isn't accepted, we had two change to ["ec2.amazonaws.com.cn"] in order to work.

Reply:
Thanks!

Looks like it would be different for a Gov Cloud too.